colin/cli/colin.py

```diff
@@ -21,7 +21,8 @@
 import click
 import six
 
-from colin.utils.cmd_tools import get_version_msg_from_the_cmd, is_rpm_installed
+from colin.utils.cmd_tools import get_version_msg_from_the_cmd, is_rpm_installed, \
+    get_version_of_the_python_package
```

Comment:
May I know where do we need `get_version_of_the_python_package`?

I cannot used it anywhere. But maybe I opverlooked something.

Reply:
will delete it, got it from cherry picking